coffee-chats/src/main/webapp/src/components/FindChatCard.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useState, ChangeEvent } from "react"
+import "./FindChatCard.css"
+import { Typography, CardActions, Card, CardContent, Button, CardHeader, createStyles, makeStyles, Theme, Collapse, Grid, MenuItem, InputLabel, Select, FormControl, FormControlLabel, Checkbox, CircularProgress } from "@material-ui/core"
```

Comment:
It's probably a nice idea to break up this line (236 characters is a bit too much)

Reply:
Yes, I was thinking so, just mindful that styleguide says imports are exempt to line-wrapping limit. But I do agree it should be wrapped here.